src/message/message-parser.js

```diff
@@ -48,17 +47,14 @@ module.exports = class MessageParser {
    *                    data: <array of strings>
    *                  }
    */
-  static parse (message) {
-    const parsedMessages = []
+  static parse (message, callback) {
     const rawMessages = message.split(C.MESSAGE_SEPERATOR)
 
     for (let i = 0; i < rawMessages.length; i++) {
       if (rawMessages[i].length > 2) {
```

Comment:
this if statement is confusing me ( theres actually a commit message associated but I can't figure out what it means =( )